src/core/src/device.rs

```diff
@@ -187,6 +187,11 @@ pub trait Device<B: Backend>: Clone {
     ) -> Result<B::BufferView, buffer::ViewError>;
 
     ///
+    fn create_texel_buffer_view(
```

Comment:
We are trying to make it fully Vulkan compatible here (at the core API level). Vulkan doesn't have a concept of `TexelBufferView`, it just has `BufferView` and `ImageView`. I believe what you are trying to achieve is in fact our `BufferView`

Reply:
Vulkan's `VkBufferView` is indeed equivalent to my `TexelBufferView`. However, in Vulkan, BufferViews are a subsection of a buffer plus a format, and are used exclusively for Texel Buffers. However, the usage of gfx's `BufferView` in`pso::DescriptorWrite::StorageBuffer` and `UniformBuffer` indicates that you intended it to be simply a subsection of a buffer, which has no particular name in Vulkan and does not need to be explicitly created, so I created a new type.